Scripts/developer_scripts/global_run_testsuite.sh

```diff
@@ -0,0 +1,265 @@
+#!/bin/bash
```

Comment:
Why the name `global_run_testsuite.sh`?

Reply:
It is just the first thing that crossed my mind when I was looking for a a name for a the script that runs the testsuite remotely and locally.

Reply:
It could be autotest_cgal_with_cmake, wichi sounds like a more logical choice when compared with run_testsuite_with_cmake.

Reply:
For the record, this morning we said: `autotest_cgal_with_ctest`